app/assets/javascripts/application/tools/newsletter_signup.js

```diff
@@ -0,0 +1,30 @@
+$(document).on("ready", function() {
+  $("form.newsletter-subscription").on("ajax:beforeSend", function() {
+  });
+
+  $("form.newsletter-subscription").on("ajax:complete", function(xhr, data, status) {
+    if (status == "success") {
+      $(this).find(".radio-inline, input").attr("disabled", true);
+      $(this).find("input, .radio-inline").css("opacity", 0.5);
```

Comment:
I think it would be better to have the opacity and background color in the css, based on the presence of the "disabled" class.